src/benchmark/benchmark_activeobjectmgr.cpp

```diff
@@ -47,6 +47,51 @@ inline void fill(server::ActiveObjectMgr &mgr, size_t n)
 
 }
 
+template <size_t N>
+void benchInsertObjects(Catch::Benchmark::Chronometer &meter)
+{
+	server::ActiveObjectMgr mgr;
+	
+	meter.measure([&] {
+		fill(mgr, N);
+	});
+
+	mgr.clear(); // implementation expects this
+}
+
+template <size_t N>
+void benchRemoveObjects(Catch::Benchmark::Chronometer &meter)
+{
+	server::ActiveObjectMgr mgr;
+	
+	fill(mgr, N);
+	meter.measure([&] {
+		mgr.clear();
+	});
+
+	mgr.clear();
+}
+
+template <size_t N>
+void benchUpdateObjectPositions(Catch::Benchmark::Chronometer &meter)
+{
+	server::ActiveObjectMgr mgr;
+	std::vector<v3f> newPositions;
+	newPositions.reserve(N);
+	
+	fill(mgr, N);
+	for (size_t i = 0; i < N; i++) {
+		newPositions.push_back(randpos());
+	}
+	meter.measure([&] {
+		size_t i = 0;
+		mgr.step(0,[&](ServerActiveObject* obj){
+			obj->setBasePosition(newPositions.at(i++));
```

Comment:
Why not directly call `randpos` here? Is accessing the vector really faster than computing 3 pseudorandom numbers? Does it make much of a difference?